### PR TITLE
MILESTONE: POPUP WINDOWS

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
# POPUP MENU

- Add popup window to mobile view
- Add popup window to desktop view
- Create a project CSS file for a popup window
- Add javascript to toggle popup window in mobile and desktop view
- Format code to use DRY principle
- Add responsiveness to project images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -230,13 +231,13 @@ <h2 class="project-name">Multi-Post Stories</h2>
...
 
-          <button type="button" class="project-button">See Project</button>
+          <button type="button" class="project-button project-button1" id="button1">See Project</button>
         </div>
       </div>
       <!--project template 1 ends here-->
 
       <!--project template 2-->
-      <div class="project-reverse">
+      <div class="project-reverse project">
```
Comment (moise10r) [inline index.html]: - [ ] Based on requirements all the content of popup modal is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie: eg:
````javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
````
Please create an object and make the modal popup data populated with the data from the projects array. 😉



If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@Team,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- The popup is responsive, it fits mobile and desktop sizes properly.✔️ 
- You have a professional README file
-  You have a descriptive PR, keep rocking :rocket: 
 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: project.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  .button-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    gap: 30px;
+    align-items: center;
+    margin-bottom: 50px;
+    margin-top: 20px;
+  }
+
```
Comment (moise10r) [inline project.css]: - [ ] The popup follows the design for both desktop and mobile version from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), Kindly add these icons to the `See live` and `See source` buttons of your popup window :arrow_down: 

Your design: 

b1


Figma:

g1

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @Amedzro-Elikplim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**